src/pages/Reviews/components/CardList.tsx

```diff
@@ -0,0 +1,80 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { useGetStoresReview } from 'queries/modules/stores/useStoresQuery';
+import Card from './Card';
+
+import { get, isUndefined } from 'lodash';
+import StarRating from 'pages/Reviews/components/StarRating';
+
+const CardList = () => {
+  const { data } = useGetStoresReview(1);
```

Comment:
저는 소소하지만 요렇게 alias로 더욱 명확한 변수명이 되면 좋을 것 같습니다!
```suggestion
  const { data: storeReview } = useGetStoresReview(1);
```

Reply:
저도 선호하는 부분인데 이번엔 놓쳤네요. 감사합니다.